gradio/helpers.py

```diff
@@ -52,7 +52,7 @@ def create_examples(
     run_on_click: bool = False,
     preprocess: bool = True,
     postprocess: bool = True,
-    api_name: str | None | Literal[False] = False,
+    api_name: str | None | Literal[False] = None,
```

Comment:
This is needed so that downstream apps that use `gr.load()` can select examples as well.

Reply:
Makes sense but there was a reason we set it to False originally but can't remember.

Reply:
I think it was because we wanted Interface to only have one named endpoint for simplicity. But I don't think that's too important and it prevents Interfaces from being `gr.load`-ed